app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,86 @@
+class MessagesController < ApplicationController
+    # before_action :correct_message, only: [:destroy]
+    before_action :set_message, only: %i[ show edit update destroy ]
+    def new
+        @message = Message.new
+    end
+
+    def show
+        @message = Message.find(params[:id])
+        
+        # respond_to do |format|
+        #     format.html 
+        #     format.xml {render :xml => @message}
+        #     # format.json  {render :json => @message}
+        # end
+    end
+    
+    def create
+        @message = Message.new(message_params)
+        @message.thumbnail.attach(params[:message][:thumbnail])
+        @message.video.attach(params[:message][:video])
+        if @message.save
+            reset_session
```

Comment:
Would you please walk me through why you call reset_session here?  What is this intended to do?

Reply:
Honestly that's just a mistake that slipped by